src/App.js

```diff
@@ -1,7 +1,48 @@
 import "./App.css";
+import productData from './data/productData'
+import Products from './Components/Products'
+import Cart from './Components/Cart'
+import Checkout from './Components/Checkout'
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
-};
+import React, { Component } from 'react'
+
+class App extends Component {
+  constructor(){
+    super()
+    this.state = {
+      items: [],
+    }
+    this.data = productData
+    this.addToCart = this.addToCart.bind(this)
```

Comment:
Because you're defining addToCart below as an arrow function, you don't need to use `.bind()` here!